.all-contributorsrc

```diff
@@ -9,6 +9,154 @@
   "badgeTemplate": "[![All Contributors](https://img.shields.io/badge/all_contributors-<%= contributors.length %>-orange.svg)](#contributors)",
   "skipCi": false,
   "contributorsSortAlphabetically": true,
+  "projectName": "the-turing-way",
+  "projectOwner": "the-turing-way",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "commitConvention": "none",
+  "commitType": "docs",
+  "types": {
+	"a11y": {
+      "symbol": "♿️",
+      "description": "Accessibility",
+	  "link": "[<%= symbol %>],"
+    },
```

Comment:
I don't like redefining all of these default types, but if it is the way to get around warnings and let's us enable `--strict` I think it is worth it.